### PR TITLE
add Fresh Components

### PR DESCRIPTION
**Pull request by hashrock**

### CODE REVIEW

**Review by lucacasonato — COMMENTED** [review]
image

**Comment by hashrock** [comment]
I noticed that Button I made was left disabled (it was using IS_BROWSER on a component that was not called from island). I will fix this as well so converted to draft for now

**Comment by ry** [comment]
Looks good!

Would it be difficult to syntax highlight the code?

What about adding a link to the home page?

File: www/components/gallery/CodeBox.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    props.code,
+    Prism.languages.typescript,
+    "typescript",
+  );
+  const onCopy = () => {
+    navigator.clipboard.writeText(props.code);
+    setCopied(true);
+  };
+
+  const [copied, setCopied] = useState(false);
```
Comment (crowlKats) [inline www/components/gallery/CodeBox.tsx:21]: shouldnt this be in an island?

Comment (hashrock) [reply www/components/gallery/CodeBox.tsx:21]: CodeBox.tsx is used in `islands/ComponentGallery.tsx`, so I think it is fine as a component.

Comment (crowlKats) [reply www/components/gallery/CodeBox.tsx:21]: ah wasnt aware that works; and yea, islands cant import islands, so this is fine as-is.

**Review by kt3k — APPROVED** [review]
LGTM. Nice work!

File: www/components/gallery/CodeBox.tsx (hunk 1 above)
Comment (kt3k) [reply www/components/gallery/CodeBox.tsx:21]: Oh, dependencies of islands can use these hooks. This sounds a tricky situation...

**Pull request merged**